cloudberry/zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/TwitterDataStoreForSQL.scala

```diff
@@ -0,0 +1,48 @@
+package edu.uci.ics.cloudberry.zion.model.impl
+
+import edu.uci.ics.cloudberry.zion.model.schema._
+
+object TwitterDataStoreForSQL {
+  val DatasetName = "twitter_ds_tweet"
+  val TimeFieldName = "create_at"
+  val TwitterSchemaForSQL = Schema("twitter_ds_tweet",
+                                      Seq(
+                                        TimeField("create_at"),
+                                        TextField("hashtags"),
```

Comment:
in cloudberry, the `TextField` often indicate there is an inverted index built on. I think the cloudberry type of the schema should be the same as the AsterixDB one. We only need to change the field name.